homeworks/es6-basics/xlDaNtelx/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
There's no need in `index.html`. You can run this file with Node.js or [jsbin.com](http://jsbin.com)